frontend/src/metabase/core/components/ColorSelector/ColorSelector.tsx

```diff
@@ -27,14 +27,15 @@ export const ColorSelector = ({
   withinPortal = true,
   ...props
 }: ColorSelectorProps) => {
-  const [opened, { toggle, close }] = useDisclosure(false);
+  const [opened, { turnOff: close, toggle }] = useToggle(false);
 
   return (
     <Popover
       withinPortal={withinPortal}
       opened={opened}
-      onClose={close}
+      onDismiss={close}
```

Comment:
I'm not sure if we want onChange or onDismiss here. It's actually not clear to me why both props exist haha

Reply:
From what I understand, `onClose` is called when the popover is closed, `onDismiss` is called when the popover will close because something likle a click outside or a keypress on Esc has been done.